packages/components/src/custom-gradient-picker/style.scss

```diff
@@ -66,7 +77,6 @@ $components-custom-gradient-picker__padding: $grid-unit-20; // 48px container, 1
 .components-custom-gradient-picker__inserter {
 	/*rtl:ignore*/
 	direction: ltr;
-	width: 100%;
```

Comment:
This was not necessary anymore, since `components-custom-gradient-picker__insert-point-dropdown` also sets the width